### PR TITLE
Pose estimation/numbering

### PR DESCRIPTION
**Pull request by JonathanLeban**

### CODE REVIEW

File: tutorials/pose_estimation/Documentation/0_ros_setup.md [hunk 1]
```diff
@@ -33,13 +33,14 @@ Ensure there are no unexpected errors.
...
 
-* **Action**: Navigate to your ROS workspace (e.g. `~/catkin_ws`). Assign the ROS IP in the `params.yaml` file as follows:
+5. Navigate to your ROS workspace (e.g. `~/catkin_ws`). Assign the ROS IP in the `params.yaml` file as follows:
 
 ```bash
 echo "ROS_IP: $(hostname -I)" > src/ur3_moveit/config/params.yaml
 ```
 
-**Note**: You can also manually assign this value by navigating to the `src/ur3_moveit/config/params.yaml` file and opening it for editing.
+**Note**: You can also manually assign this value by navigating to the `src/ur3_moveit/config/params.yaml` file and opening it for
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/0_ros_setup.md:42]: Should this note also use the `>` syntax?

File: tutorials/pose_estimation/Documentation/1_set_up_the_scene.md [hunk 2]
```diff
@@ -32,13 +30,13 @@ cd Unity-Robotics-Hub/tutorials/pose_estimation
 ./submodule.sh
 ```
 
-**Action**: [Install Unity `2020.2.*`.](install_unity.md)
+2. [Install Unity `2020.2.*`.](install_unity.md)
 
 
 ### <a name="step-1">Step 1: Create a New Project</a>
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/1_set_up_the_scene.md:36]: Since we are now numbering the actions, let's no longer number the steps (to match the rest of this repo). With both its too many numbers.

File: tutorials/pose_estimation/Documentation/0_ros_setup.md (hunk 1 above)
Comment (JonathanLeban) [reply tutorials/pose_estimation/Documentation/0_ros_setup.md:42]: I tried but then when we have a note and a code above or below it is really to pleasant visually

File: tutorials/pose_estimation/Documentation/1_set_up_the_scene.md [hunk 3]
```diff
@@ -65,7 +63,7 @@ Installing the different packages may take some time (few minutes).
 
 
 #### Install Dependencies
-**Action**: Install the following packages with the provided git URLs:
+4. Install the following packages with the provided git URLs:
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/1_set_up_the_scene.md:66]: Since we used numbers up above in a different way before, we need to change this. Maybe make 1., 2, 3, bullet points (since they are not actions), and then start the numbering here at 1.

File: tutorials/pose_estimation/Documentation/1_set_up_the_scene.md (hunk 2 above)
Comment (JonathanLeban) [reply tutorials/pose_estimation/Documentation/1_set_up_the_scene.md:36]: yes you are right :)

File: tutorials/pose_estimation/Documentation/1_set_up_the_scene.md (hunk 3 above)
Comment (sarahwolf32) [reply tutorials/pose_estimation/Documentation/1_set_up_the_scene.md:66]: Actually, let's do 1, 2, 3 for the 3 packages below.

File: tutorials/pose_estimation/Documentation/0_ros_setup.md (hunk 1 above)
Comment (sarahwolf32) [reply tutorials/pose_estimation/Documentation/0_ros_setup.md:42]: You're saying it looks weird to sandwhich code between two notes? Then why not put both notes beneath the code?

**Comment by sarahwolf32** [comment]
Names of the links need to be updated in the data collection section of quick_demo_train.md  - since we removed the step numbering.

File: tutorials/pose_estimation/Documentation/quick_demo_full.md [hunk 4]
```diff
@@ -15,32 +15,31 @@ If you just want to run the completed project, this section can help you get up
 
 You will first need to **clone** this repository. This project has dependencies on a few git submodules, so you'll need to include the flag `--recurse-submodules` in your clone command. 
 
-**Note For Windows users**:
-
+>Note For Windows Users:
 You need to have a software enabling you to run bash files. One option is to download [GIT](https://git-scm.com/downloads). During installation of GIT, add GIT Bash to windows context menu by selecting its option. After installation right click in your folder select GIT Bash Here (see attached pic).
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/quick_demo_full.md:19]: `GIT Bash Here (see attached pic)` needs link.

File: tutorials/pose_estimation/Documentation/quick_demo_train.md [hunk 5]
```diff
@@ -5,33 +5,32 @@
 To follow this tutorial you need to **clone** this repository even if you want to create your Unity project from scratch. In fact, we need the ROS folder of the repository which is dependent of gitsubmodules (hence the command `--recurse-submodules`). 
 
 
-**Note For Windows users**:
-
+>Note For Windows Users:
 You need to have a software enabling you to run bash files. One option is to download [GIT](https://git-scm.com/downloads). During installation of GIT, add GIT Bash to windows context menu by selecting its option. After installation right click in your folder select GIT Bash Here (see attached pic).
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/quick_demo_train.md:9]: GIT Bash Here (see attached pic) needs link.

File: tutorials/pose_estimation/Documentation/quick_demo_train.md [hunk 6]
```diff
@@ -5,33 +5,32 @@
...
+2. The size of the images that will be used for pose estimation depends on a setting in the Game view. Select the `Game` view and select `Free Aspect`. Then select the **+**, with the message `Add new item` on it if you put your mouse over the + sign. For the Width select `650` and for the Height select `400`. A gif below shows you how to do it. 
 
 <p align="center">
 <img src="Gifs/2_aspect_ratio.gif"/>
 </p>
 
 ## <a name="data-collection">Data Collection</a>
 The project is set up for data collection by default, so you can get started without too much work. To see how,
-follow the instructions in [phase 3: step 1](3_data_collection_model_training.md#step-1-collect-the-training-and-validation-data) of the tutorial. This section will explain how to set the random seed of the environment, choose how many training data examples you'd like to collect, and get it running. 
+follow the instructions in [phase 3: step 1](3_data_collection_model_training.md#step-1) of the tutorial. This section will explain how to set the random seed of the environment, choose how many training data examples you'd like to collect, and get it running.
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/quick_demo_train.md:32]:
```suggestion
follow the instructions in [Phase 3: Collect the Training and Validation Data](3_data_collection_model_training.md#step-1) of the tutorial. This section will explain how to set the random seed of the environment, choose how many training data examples you'd like to collect, and get it running. 
```

File: tutorials/pose_estimation/Documentation/quick_demo_train.md [hunk 7]
```diff
@@ -5,33 +5,32 @@
...
 <img src="Gifs/2_aspect_ratio.gif"/>
 </p>
 
 ## <a name="data-collection">Data Collection</a>
 The project is set up for data collection by default, so you can get started without too much work. To see how,
-follow the instructions in [phase 3: step 1](3_data_collection_model_training.md#step-1-collect-the-training-and-validation-data) of the tutorial. This section will explain how to set the random seed of the environment, choose how many training data examples you'd like to collect, and get it running. 
+follow the instructions in [phase 3: step 1](3_data_collection_model_training.md#step-1) of the tutorial. This section will explain how to set the random seed of the environment, choose how many training data examples you'd like to collect, and get it running. 
 
-If you'd like to then move on to training a pose estimation model on the data you've collected, move on to [phase 3: step 2](3_data_collection_model_training.md#step-2-train-the-deep-learning-model). 
+If you'd like to then move on to training a pose estimation model on the data you've collected, move on to [phase 3: step 2](3_data_collection_model_training.md#step-2).
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/quick_demo_train.md:34]:
```suggestion
If you'd like to then move on to training a pose estimation model on the data you've collected, move on to [Phase 3: Train the Deep Learning Model](3_data_collection_model_training.md#step-2). 
```

File: tutorials/pose_estimation/Documentation/0_ros_setup.md [hunk 8]
```diff
@@ -2,27 +2,27 @@
...
+> ```
 
-```bash 
-sudo pip3 install rospkg numpy jsonpickle scipy easydict torch==1.7.1 torchvision==0.8.2 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html
-```
 
 Most of the ROS setup has been provided via the `ur3_moveit` package. This section will describe the provided files.
 
-* **Action**: If you have not already built and sourced the ROS workspace since importing the new ROS packages, navigate to your ROS workplace, and run 
+4. If you have not already built and sourced the ROS workspace since importing the new ROS packages, navigate to your ROS workplace, and run
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/0_ros_setup.md:25]:
```suggestion
4. If you have not already built and sourced the ROS workspace since importing the new ROS packages, navigate to your ROS workplace, and run: 
```

File: tutorials/pose_estimation/Documentation/0_ros_setup.md [hunk 9]
```diff
@@ -49,7 +50,7 @@ e.g.
 ```yaml
 ROS_IP: 10.0.0.250
 ```
-**Note**: Learn more about the server endpoint and ROS parameters [here](https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/main/tutorials/ros_unity_integration/server_endpoint.md).
+    > Note: Learn more about the server endpoint and ROS parameters [here](https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/main/tutorials/ros_unity_integration/server_endpoint.md).
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/0_ros_setup.md:53]: This formatting is broken for some reason, the ">" is showing up in the markup.

Comment (sarahwolf32) [reply tutorials/pose_estimation/Documentation/0_ros_setup.md:53]: And there's another *Note* to fix on line 67.

File: tutorials/pose_estimation/Documentation/1_set_up_the_scene.md [hunk 10]
```diff
@@ -17,11 +17,9 @@ In this first phase of the tutorial, we will start by downloading and installing
 To follow this tutorial you need to **clone** this repository even if you want to create your Unity project from scratch. In fact, we need the ROS folder of the repository which is dependent of gitsubmodules (hence the command `--recurse-submodules`). 
 
 
-**Note For Windows users**:
+  > Note For Windows users: You need to have a software enabling you to run bash files. One option is to download [GIT](https://git-scm.com/downloads). During installation of GIT, add GIT Bash to windows context menu by selecting its option. After installation, right click in your folder, and select `GIT Bash Here`.
```
Comment (sarahwolf32) [inline tutorials/pose_estimation/Documentation/1_set_up_the_scene.md:20]: `GIT Bash Here` needs link. I thought we used to have one? Not sure what happened.

**Review by sarahwolf32 — APPROVED** [review]
